examples/lib/defaultHtmlStepUi.js

```diff
@@ -308,7 +308,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
       return output.split('/')[1].split(';')[0];
     }
 
-    $stepAll('.download-btn').on('click', () => {
+    $stepAll('.download-btn').unbind().on('click', () => {
```

Comment:
Wouldn't it be better to use `.off('click')` instead of `unbind`? `unbind` will remove all the event handlers but we only want the click event handler to be removed.

Reply:
Also, did you find out why `$stepAll` is used here? This is a bit confusing. Does it work with just `$step`? If yes, then it would be better to use `$step` instead of `$stepAll`.

Wait, the error could be caused *because* of `$stepAll`. We want to add a download listener to the current step but `$stepAll` will add a listener to every step each time a new step is added. So every time a step is added, an extra listener will be added to the older steps. If I am not wrong then changing `$stepAll` to `$step` without `unbind` or `off` should fix everything.

Reply:
Changing it to `$step` didn't work. 
Isn't `$stepAll` derived from `scopeSelectorAll` which should basically mean to select all the instances of the element present inside the scope (i.e in the current step). By that shouldn't `$stepAll` refers to the all the elements in the current step.

Reply:
Oh yeah! I guess I forgot myself. Lol. I was the one who added `scopeQuery` :sweat_smile:.
Okay then your code should work fine. Will review again in a min.

Reply:
Should I revert back to off(‘click) in this one?

Reply:
Yeah, I think you should. Sorry for the late response.

Reply:
Done